### PR TITLE
 Applied second batch of enhancements for 6.0

### PR DESCRIPTION
**Pull request by julian-cable**

### CODE REVIEW

File: en-US/Design.xml [hunk 1]
```diff
@@ -702,7 +702,7 @@ $ vi myFile.txt
 	 	<section id="refer-object-names">
 			<title>Referring to Object Names</title>
 			 <para>
-				Do not use object names as part of normative grammar. A sentence should be complete without the object name.
+				When using object names, write a sentence so that it would be complete without the object name.
```
Comment (nmuller66) [inline en-US/Design.xml:705]: Instead of "it would be complete" how about "is complete"

File: en-US/Language.xml [hunk 2]
```diff
@@ -1054,7 +1068,14 @@ Numbers (move from N entry) -->
 								</row>
 								 <row>
 									<entry> To create another administrator, click <guimenuitem>New</guimenuitem> on the <guimenuitem>File</guimenuitem> menu. </entry>
```
Comment (nmuller66) [inline en-US/Language.xml:1070]: A little confused, this seems identical to line 1073, not sure what I'm mnissing.

**Review by nmuller66 — COMMENTED** [review]
@julian-cable left a couple of comments

File: en-US/Design.xml (hunk 1 above)
Comment (julian-cable) [reply en-US/Design.xml:705]: Fixed

File: en-US/Language.xml (hunk 2 above)
Comment (julian-cable) [reply en-US/Language.xml:1070]: I assume that the distinction here is between administrator as an account versus as a person.

**Pull request merged**